### PR TITLE
Allow configuring the shared runtime via an environment variable

### PR DESCRIPTION
**Pull request by pranavkm**
cc @anurse \ @BrennanConroy

### CODE REVIEW

File: build/KoreBuild.sh [hunk 1]
```diff
@@ -40,6 +40,8 @@ version=$(<$versionFile)
 
 [ -z "$KOREBUILD_DOTNET_CHANNEL" ] && KOREBUILD_DOTNET_CHANNEL=rel-1.0.0
 [ -z "$KOREBUILD_DOTNET_VERSION" ] && KOREBUILD_DOTNET_VERSION=$version
+[ -z "$KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION " ] && KOREBUILD_DOTNET_VERSION=1.0.0
```
Comment (BrennanConroy) [inline build/KoreBuild.sh]: These need to be the same variable...

Comment (BrennanConroy) [reply build/KoreBuild.sh]: Or wait, we don't have a shared runtime install here?

Comment (pranavkm) [reply build/KoreBuild.sh]: Fixed

**Comment by BrennanConroy** [comment]
:shipit:

**Pull request merged**